traffic_ops/testing/api/v14/jobs_test.go

```diff
@@ -103,3 +126,44 @@ func GetTestJobs(t *testing.T) {
 		}
 	}
 }
+
+func GetTestInvalidationJobs(t *testing.T) {
+	jobs, _, err := TOSession.GetInvalidationJobs(nil, nil)
+	if err != nil {
+		t.Fatalf("error getting invalidation jobs: %v\n", err)
+	}
+
+	toDSes, _, err := TOSession.GetDeliveryServices()
+	if err != nil {
+		t.Fatalf("cannot GET DeliveryServices: %v - %v\n", err, toDSes)
+	}
+
+	dsIDNames := map[uint]string{}
+	for _, ds := range toDSes {
+		if ds.ID <= 0 {
+			t.Fatalf("Erroneous Delivery Service - has invalid ID: %+v\n", ds)
+		}
+		dsIDNames[uint(ds.ID)] = ds.XMLID
```

Comment:
is this map used all at?

Reply:
I think I planned to at one point, but no, it seems not.